src/go/cmd/token-vendor/repository/k8s/k8s.go

```diff
@@ -63,21 +65,22 @@ func (k *K8sRepository) ListAllDeviceIDs(ctx context.Context) ([]string, error)
 // The public key is stored under a specific key in the configmap. If the configmap
 // does not exist, we return an empty string. For any other error or a malformed
 // configmap we return an error.
-func (k *K8sRepository) LookupKey(ctx context.Context, deviceID string) (string, error) {
+func (k *K8sRepository) LookupKey(ctx context.Context, deviceID string) (string, string, error) {
```

Comment:
I think we should rename this to LookupConfig and return a config struct here. However that can be a follow-up.

Reply:
Done